docs/Using-Fleet/MDM-setup.md

```diff
@@ -7,41 +7,89 @@ In Fleet, MDM features are supported for Macs running macOS 12 (Monterey) and hi
+2. Under **Apple Push Certificates Portal**, select "Request," then fill out the form. This should generate three files and send an email to you with an attached CSR file.
 
 `fleetctl` CLI:
 
-1. Run `fleetctl generate mdm-apple --email <email> --org <org>`.
+1. Run `fleetctl generate mdm-apple --email <email> --org <org>`. This should download three files and send an email to you with an attached CSR file.
+
+### Step 2: generate an APNs certificate from APNs
+
+1. Log in to or enroll in [Apple Push Certificates Portal](https://identity.apple.com).
+2. Select **Create a Certificate**
+3. Upload your CSR and input a friendly name, such as "Fleet MDM."
+4. Download the APNs certificate
+
+> Take note of the Apple ID you use to sign into Apple Push Certificates Portal. You'll need to use the same Apple ID when renewing your APNs certificate.
+
+### Step 3: configure Fleet with the required files
+
+With the four generated files, we now give them to the Fleet server. 
+
+Restart the Fleet server with the contents of the APNs certificate, APNs private key, SCEP certificate, and SCEP private key in following environment variables:
+* [FLEET_MDM_APPLE_APNS_CERT_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-apns-cert-bytes)
+* [FLEET_MDM_APPLE_APNS_KEY_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-apns-key-bytes)
+* [FLEET_MDM_APPLE_SCEP_CERT_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-scep-cert-bytes)
+* [FLEET_MDM_APPLE_SCEP_KEY_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-scep-key-bytes)
+
+> You do not need to provide the APNs CSR which was emailed to you. 
+
+Confirm that Fleet is set up by visitng the "Fleet UI" or using "fleetctl."
+
+Fleet UI:
+
+1. Head to the **Settings > Integrations > Mobile device management (MDM)** page.
+
+2. Look at the **Apple Push Certificates Portal** section.
 
-2. Follow the on-screen instructions.
+`fleetctl` CLI:
+
+1. Run `fleetctl get mdm-apple`.
 
-> Take note of the Apple ID you use to sign into Apple Push Certificates Portal. You'll need to use the same Apple ID when renewing your APNs certificate. Apple requires that APNs certificates are renewed once every year. To renew, see the [APNs Renewal section](#apns-renewal) .
+You should see information about the APNs certificate such as serial number and renewal date. 
 
 ## Renewing APNs
 
-Apple requires that APNs certificates are renewed once every year. You can see the certificate's renewal date and other important APNs information using the Fleet UI or the `fleetctl` command-line interface:
+> Apple requires that APNs certificates are renewed once every year. 
+> * Be sure to do it early. If you renew after a certificate has expired, you will have to turn MDM off and back on for all macOS hosts. 
+> * Be sure to use the same Apple ID from year-to-year. If you don't, you will have to unenroll and re-enroll all macOS hosts. 
```

Comment:
```suggestion
> * Be sure to use the same Apple ID from year-to-year. If you don't, you will have to turn MDM off and back on for all macOS hosts.
```

I think we should be consistent about using "turn on/off" for MDM instead of "enroll/unenroll." 

In Fleet, turn on/off is for MDM enrollment, enroll/unenroll is for osquery enrollment. 

All hosts, regardless of MDM features, enroll to Fleet.

What do you think?

Reply:
Yikes. I see the case there. However, I think elsewhere in our app, and maybe more difficult in 3rd party language, it says "enroll/unenroll." Need to think on this.